cm2350/e200z7.py

```diff
@@ -137,8 +136,13 @@ def __init__(self, vw):
         # class the PPC_e200z7 is designed so that the core vivisect workspace
         # emulator can be removed in the future to improve performance (at the
         # cost of analysis/inspection/live debug capabilities).
+        vimp_emu.WorkspaceEmulator.__init__(self, self.vw, nostack=True, funconly=False)
 
-        vimp_emu.WorkspaceEmulator.__init__(self, vw, nostack=True, funconly=False)
+        # The workspace emulator initializer copies the memory maps from the 
+        # vivisect workspace, but the standalone emulator memory maps should be 
+        # independent from the vivisect workspace.
+        for mva, _, _, _ in self.vw.getMemoryMaps():
+            self.delMemoryMap(mva)
```

Comment:
ideally, yes. I haven't come up with a good way to merge the vivisect workspaces with the MMIO capable memory regions in the emulator yet. This set of changes was intended to try and start taking more advantage of the vivisect capabilities for parsing binaries. the challenge is how best to merge those with things like the MPC5674F's FLASH peripheral. since the memory maps in the workspace probably won't be exactly the same size as the flash region.

That's just an example. I'm more than happy to talk about it. this particular workaround is intended to be just a workaround for now until we figure out how best to handle this.